lib/containers/runtime.pm

```diff
@@ -65,20 +116,21 @@ sub up {
     my $mode           = $args{daemon}         ? '-d'    : '-it';
```

Comment:
This function does not make sense. It should be `run()` not `up()`.

Reply:
it had approved before. and it is not in space of this refactoring. lets keep the review simple otherwise this will be never come to an end